src/components/Project/ProjectTable.js

```diff
@@ -21,6 +27,25 @@ export default class ProjectTable extends Component {
       })
       .catch(error => console.log(error));
   }
+
+  componentDidMount() {
+    this.getProjectData();
+  }
+
+  deleteOnClick(projectId) {
+    const options = {
```

Comment:
https://screencast.com/t/JNaFm0Ede6 - need to fix CORS problem, deletion doesn't work  for me